src/crypto.ts

```diff
@@ -195,4 +200,41 @@ export default class Crypto {
       decodeBase64(formSecretKey)
     )
   }
+
+  /**
+   * Download and decrypt the given attachments.
+   * @param formSecretKey Secret key as a base-64 string
+   * @param decryptParams The params containing encrypted content and information.
+   * @returns The decrypted attachments if successful. Else, null will be returned.
+   * @throws {MissingPublicKeyError} if a public key is not provided when instantiating this class and is needed for verifying signed content.
+   */
+  downloadAndDecryptAttachments = async(formSecretKey: string, decryptParams: DecryptParams): Promise<DecryptedAttachments | null> => {
```

Comment:
I'm wondering if this function should accept a `DecryptedContent` rather than a `DecryptParams`, so we don't force the user to decrypt the response content twice. wdyt?

Reply:
After discussion, we are going with `decryptWithAttachment`